modules/gdscript/tests/test_completion.h

```diff
@@ -31,6 +31,8 @@
 #ifndef TEST_COMPLETION_H
 #define TEST_COMPLETION_H
 
+#include "core/io/resource_loader.h"
+#include "core/io/resource_uid.h"
```

Comment:
I'd put these among the rest, none of the tests are run without tools anyway so no need for includes